traits/WhereTrait.php

```diff
@@ -47,6 +47,17 @@ private function initWhereOr():void {
             }
         }
 
+        /**
+         * Initiates string starter for first call of 'where' method
+         *
+         * @return void
+         */
+        private function initWhere():void {
+            if ($this->where === "") {
+                $this->where = "WHERE";
+            }
+        }
```

Comment:
The initialization of the string was extracted similar to 'whereAnd' and 'whereOr'.